platforms/gke/base/core/README.md

```diff
@@ -2,36 +2,28 @@
 
 ## Pull the source code
 
-1. Open [Cloud Shell](https://cloud.google.com/shell).
```

Comment:
I would keep this, unless we want to have a section about which kind of shell is required, which tools (besides Terraform) we need (bash, sed, docker....)